tests/unit/test_orgs_invites.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from github3 import GitHubError
+from github3.orgs import Organization
+
+from . import helper
+
+url_for = helper.create_url_helper('https://api.github.com/orgs/github')
+
+get_org_example_data = helper.create_example_data_helper(
+    'org_example')
```

Comment:
If this doesn't fit on the previous line, then we prefer to have the closing `)` on the next dedented.